packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.targets

```diff
@@ -16,7 +17,9 @@
   <Target Name="CheckForDuplicatePageItems" 
           BeforeTargets="_CheckForInvalidConfigurationAndPlatform;CoreCompile"
           DependsOnTargets="CheckForDuplicateItems"
-          Condition="('$(UseWPF)' == 'true') And ('$(_TargetFrameworkVersionWithoutV)' != '') And ('$(_TargetFrameworkVersionWithoutV)' >= '$(_WindowsDesktopSdkTargetFrameworkVersionFloor)')">
+          Condition="('$(UseWPF)' == 'true') And 
+                     ('@(_WindowsDesktopTargetFrameworkVersion)' != '$(_InvalidTargetFrameworkVersion)') And 
+                     ('@(_WindowsDesktopTargetFrameworkVersion)' >= '$(_WindowsDesktopSdkTargetFrameworkVersionFloor)')">
```

Comment:
These targets will only run for inner builds so it's not strictly necessary to change the property references here, but I agree that you should for consistency.